source/migration-guide/index.rst

```diff
@@ -12,7 +12,7 @@ From Wazuh 4.0.0 to Wazuh 4.2.7, the default Wazuh installation included the Waz
 
 - :doc:`Migrating to the Wazuh indexer </migration-guide/wazuh-indexer>`: Follow this section to migrate from Open Distro for Elasticsearch 1.13 to the Wazuh indexer. This new component consists of a distribution of `Opensearch <https://github.com/opensearch-project/OpenSearch>`_ with additional tools that Wazuh has created to assist with the installation and configuration of the search engine. 
 
-- :doc:`Migrating to the Wazuh dashboard </migration-guide/wazuh-dashboard>`: This section will guide you through the migration from Open Distro for Elasticsearch Kibana 1.13 to the Wazuh dashboard. This new web interface for the Wazuh platform is a customized `OpenSearch Dashboards <https://github.com/opensearch-project/OpenSearch-Dashboards>`_ distribution that includes the `Wazuh plugin <https://github.com/wazuh/wazuh-kibana-app>`_.
+- :doc:`Migrating to the Wazuh dashboard </migration-guide/wazuh-dashboard>`: This section will guide you through the migration from Open Distro for Elasticsearch Kibana 1.13 to the Wazuh dashboard. This new web interface for the Wazuh platform is a customized `OpenSearch Dashboards <https://github.com/opensearch-project/OpenSearch-Dashboards>`_ distribution that includes the `Wazuh plugin <https://github.com/wazuh/wazuh-dashboard-plugins>`_.
```

Comment:
Should we refer to it as `Wazuh plugin` or `Wazuh App`  or how? We need to be consistent throughout the documentation.

Reply:
We'll work on this in https://github.com/wazuh/internal-devel-requests/issues/391